vizro-core/mkdocs.yml

```diff
@@ -5,22 +5,33 @@ nav:
   - Get started:
       - Quickstart: pages/tutorials/first_dashboard.md
       - Explore Vizro: pages/tutorials/explore_components.md
-  - User Guide:
-      - Install: pages/user_guides/install.md
-      - Themes: pages/user_guides/themes.md
-      - Dashboards: pages/user_guides/dashboard.md
-      - Pages: pages/user_guides/pages.md
-      - Navigation: pages/user_guides/navigation.md
-      - Charts/Components: pages/user_guides/components.md
-      - Layouts: pages/user_guides/layouts.md
-      - Controls: pages/user_guides/control.md
-      - Custom Charts: pages/user_guides/custom_charts.md
-      - Custom Components: pages/user_guides/custom_components.md
-      - Data: pages/user_guides/data.md
-      - Actions: pages/user_guides/actions.md
-      - Assets: pages/user_guides/assets.md
-      - Integrations: pages/user_guides/integration.md
-      - Run Methods: pages/user_guides/run.md
+  - User Guides:
+      - Fundamentals:
+          - Install: pages/user_guides/install.md
+          - Pages: pages/user_guides/pages.md
+          - Dashboards: pages/user_guides/dashboard.md
```

Comment:
Does it make sense to you that the 'Dashboards' come before the 'Pages' section?

Reply:
Sorry there was a wrong base, this is outdated

Reply:
It was actually on a different PR, but I can change it quickly